stm32-metapac/build.rs

```diff
@@ -40,11 +40,22 @@ pub struct Peripheral {
     pub block: Option<String>,
     #[serde(default)]
     pub clock: Option<String>,
+    #[serde(default)]
+    pub pins: Option<Vec<Pin>>,
+}
+
+#[derive(Debug, Eq, PartialEq, Clone, Deserialize)]
+pub struct Pin {
+    pub pin: String,
+    pub signal: String,
+    pub af: Option<String>,
 }
 
 struct BlockInfo {
-    module: String,  // usart_v1/USART -> usart
-    version: String, // usart_v1/USART -> v1
+    module: String,
```

Comment:
accidental reformat? if rustfmt insists on moving it, put it on top of the field with `///`